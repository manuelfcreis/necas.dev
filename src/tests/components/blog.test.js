import React from "react";
import Blog from "../../components/blog";
import { shallow } from "enzyme";

it("renders correctly", () => {
  const element = shallow(<Blog open={true} />);
  expect(element).toMatchSnapshot();
});
