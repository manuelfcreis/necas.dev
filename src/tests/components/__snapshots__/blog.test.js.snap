// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Card
  classes="container"
>
  <div
    className="pagination"
  />
</Card>
`;
